src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -53,18 +55,19 @@ public AddCommand parse(String args) throws ParseException {
         }
 
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_NRIC, PREFIX_DOB, PREFIX_GENDER,
-                PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS);
-        Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
-        Nric nric = ParserUtil.parseNric(argMultimap.getValue(PREFIX_NRIC).get());
-        DateOfBirth dob = ParserUtil.parseDateOfBirth(argMultimap.getValue(PREFIX_DOB).get());
-        Gender gender = ParserUtil.parseGender(argMultimap.getValue(PREFIX_GENDER).get());
-        Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
-        Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
-        Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+                PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_PRIORITY);
+        Name name = Name.parseName(argMultimap.getValue(PREFIX_NAME).get());
+        Nric nric = Nric.parseNric(argMultimap.getValue(PREFIX_NRIC).get());
+        DateOfBirth dob = DateOfBirth.parseDateOfBirth(argMultimap.getValue(PREFIX_DOB).get());
+        Gender gender = Gender.parseGender(argMultimap.getValue(PREFIX_GENDER).get());
+        Phone phone = Phone.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
+        Email email = Email.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
+        Address address = Address.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = new HashSet<>();
         Set<Appointment> appointmentList = Collections.emptySet();
         Set<MedCon> medConList = Collections.emptySet();
-        Priority priority = new Priority();
+        // TODO: change Magic String "NONE" to valid enum
```

Comment:
Maybe liaise with @PresleyChew abt this since he created priority?

Reply:
We are not giving the option to assignPriority via adding patient, as such no need to parse for Priority when adding